src/modm/board/disco_f051r8/board.hpp

```diff
@@ -38,7 +38,7 @@ struct SystemClock
 		// enable internal 8 MHz HSI RC clock
 		Rcc::enableInternalClock();
 		// (internal clock / 2) * 12 = 48MHz
-		Rcc::enablePll(Rcc::PllSource::InternalClock, 12, 1);
+		Rcc::enablePll(Rcc::PllSource::InternalClock, {12, 1});
```

Comment:
Hmmm, this was not my original intention in #361.

```cpp
// old
Rcc::enablePll(Rcc::PllSource::InternalClock, 12, 1);
```
```cpp
// More verbose and explicit which PLL factors are set
Rcc::PllFactors pllFactors{
    .pllM = 12,
    .pllN = 1,
};
Rcc::enablePll(Rcc::PllSource::InternalClock, pllFactors);
```

Reply:
Yeah, the implicit construction doesn't work anyways, due to overloading issues. I'll make it explicit then.